stable/README.md

```diff
@@ -1,167 +1,167 @@
-$ mv helm /usr/local/bin
-$ mv tiller /usr/local/bin
 ```
 
-If you're running on Mac, the curl commands above would be:
-
+If you're running on Mac, the commands are:
 ```bash
-$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.14.1-darwin-amd64.tar.gz | tar xz
+$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.16.1-darwin-amd64.tar.gz | tar xz
 $ cd darwin-amd64
+```
+
+Move the Helm binaries to /usr/local/bin
+```
 $ mv helm /usr/local/bin
 $ mv tiller /usr/local/bin
 ```
 
-The following command will configure the current users environment for Helm; it will create a `.helm` directory in `${HOME}`.
+Confirm you can run the Helm client: `helm help`.
 
-```bash
-$ helm init --client-only
-$HELM_HOME has been configured at /home/clusteradmin/.helm.
-Not installing Tiller due to 'client-only' flag having been set
-Happy Helming!
-```
 
-Alternatively, to keep isolated between individuals sharing a login, you can have separate Kube, Helm, and Tiller state (so run these before the above init command):
+## Create the _nuodb_ namespace to install NuoDB
 
-```bash
-export TILLER_NAMESPACE=nuodb
-export HELM_HOME=`pwd`/.helm
-export KUBECONFIG=`pwd`/.kube/config
+```
+kubectl create namespace nuodb
 ```
 
-### Install the Tiller Server
-
-We will be creating the Tiller server in the `kube-system` namespace so that it is available to all projects.
+## Create the _tiller_ service account in the _kube-system_ namespace 
 
-```bash
-oc project kube-system
 ```
-
-Create a new service account for tiller.
-```bash
-kubectl -n kube-system create serviceaccount tiller-system
+kubectl -n kube-system create serviceaccount tiller
 ```
 
-Give `cluster-admin` permissions to the newly created service account.
+## Assign the _cluster-admin_ role to the _tiller_ service account
 ```bash
-kubectl create clusterrolebinding tiller-system \
+kubectl create clusterrolebinding tiller \
 --clusterrole cluster-admin \
---serviceaccount=kube-system:tiller-system
+--serviceaccount=kube-system:tiller
 ```
 
-Start the Tiller server.
-```bash
-helm init --service-account tiller-system --tiller-namespace kube-system
-```
+Tiller is now available to all projects.
 
-We’ll check that the Helm client and Tiller server are able to communicate correctly by running helm version. The results should be as follows:
+## For Kubernetes
 
-```bash
-$ helm version
-Client: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
-Server: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
-```
+Start Helm and Tiller 
 
-You can also verify that the Tiller server is running via `kubectl`.
-```bash
-kubectl get pods -n kube-system
-NAME                                READY   STATUS    RESTARTS   AGE
-...
-tiller-deploy-8c5679674-k9c7m       1/1     Running   0          47m
-...
+```
+$ helm init --service-account tiller --tiller-namespace kube-system
```

Comment:
is this really required? I usually just use `helm init`. Have you verified these steps?

Reply:
yes, this is what I used.  It works. It's also modeled after many tiller installs on the internet for K8s.